#goal
geometry_msgs/Pose2D pose_dest
---
#result
geometry_msgs/Pose2D pose_final
---
#feedback
geometry_msgs/Pose2D pose_mid
